libs/jobs/src/lib/__tests__/app/config/root.config.ts

```diff
@@ -1,19 +1,15 @@
 import { JobsConfig } from '../../../config';
-import { Type } from 'class-transformer';
-import { ValidateNested } from 'class-validator';
 import { DbConfig } from '@softkit/typeorm';
 import { LoggerConfig } from '@softkit/logger';
+import { ValidateNestedProperty } from '@softkit/validation';
 
 export class RootConfig {
-  @ValidateNested()
-  @Type(() => JobsConfig)
+  @ValidateNestedProperty({ required: false, classType: JobsConfig })
   jobsConfig!: JobsConfig;
 
-  @ValidateNested()
-  @Type(() => DbConfig)
+  @ValidateNestedProperty({ required: false, classType: DbConfig })
   db!: DbConfig;
 
-  @ValidateNested()
-  @Type(() => LoggerConfig)
+  @ValidateNestedProperty({ required: false, classType: LoggerConfig })
```

Comment:
if it's annotated with `!` then it's required and I believe it has to be required for all